modules/op-supported-parameters-tekton-chains-configuration.adoc

```diff
@@ -61,6 +61,11 @@ Cluster administrators can use various supported parameter keys and values to co
 | The signature backend for signing pipeline run payloads.
 | `x509`, `kms`
 | `x509`
+
+| `artifacts.pipelinerun.enable-deep-inspection`
+| When this parameter is `true`, {tekton-chains} records the results of the child task runs of a pipeline run. When this parameter is `false`, {tekton-chains} records the results of the pipeline run but not of its child task runs.
+| `x509`, `kms`
```

Comment:
```suggestion
| `"true", "false"`
```

For ref: https://github.com/tektoncd/chains/blob/main/docs/config.md#pipelinerun-configuration